EIPS/eip-1328.md

```diff
@@ -28,26 +25,44 @@ WalletConnect request URI with the following parameters:
     version       = 1*DIGIT
     parameters    = parameter *( "&" parameter )
     parameter     = key "=" value
-    key           = "bridge" / "key"
+    key           = STRING
     value         = STRING
 
 ### Semantics
 
-Required parameters are dependent on the Walletconnect protocol version which currently includes the `key`, hex string of symmetric key, and `bridge`, encoded url of the bridge used for establishing the connection.
+Required parameters are dependent on the Walletconnect protocol version:
+
+For WalletConnect v1.0 protocol (`version`=`1`) the parameters are:
+- `key` - symmetric key used for encryption
+- `bridge` - url of the bridge server for relaying messages
+
+For WalletConnect v2.0 protocol (`version`=`2`) the parameters are:
+- `symKey` - symmetric key used for encrypting messages over relay
+- `relay-protocol` - transport protocol for relaying messages
+- `relay-data` - (optional) transport data for relaying messages
+
 
 ### Example
 
 ```
+# 1.0
 wc:8a5e5bdc-a0e4-4702-ba63-8f1a5655744f@1?bridge=https%3A%2F%2Fbridge.walletconnect.org&key=41791102999c339c844880b23950704cc43aa840f3739e365323cda4dfa89e7a
+
+# 2.0
+wc:c9e6d30fb34afe70a15c14e9337ba8e4d5a35dd695c39b94884b0ee60c69d168@2?relay-protocol=waku&symKey=7ff3e362f825ab868e20e767fe580d0311181632707e7c878cbeca0238d45b8b
 ```
 
 ## Rationale
 
 The need for this ERC stems from the discussion to move away from JSON format used in the alpha version of the WalletConnect protocol which makes for very inneficient parsing of the intent of the QR code, making it easier to create better QR code parsers APIs for Wallets to implement. Also by using a URI instead of a JSON inside the QR-Code the Android Intent system can be leveraged.
```

Comment:
This "Rationale" section reads more like it should belong in the "Motivation" section.

"Motivation" explains why the EIP as a whole needs to exist, while "Rationale" should explain individual choices made in the EIP itself.